sample.tab

```diff
@@ -1,11 +0,0 @@
-# crontab that controls running this proc
```

Comment:
I think it's a good idea to leave the file used to generate the screenshot here. So any new features have the file instantly handy?